src/App.jsx

```diff
@@ -1,12 +1,26 @@
-import { useState } from 'react'
 import './App.css'
+import { useState } from 'react'
+import { Route, Routes } from "react-router-dom"
+
+
+import Header from './components/Header'
+import Home from './components/Home'
+import Topics from './components/Topics'
+import Users from './components/Users'
+import Search from './components/Search'
+
 
 function App() {
-  const [count, setCount] = useState(0)
 
   return (
     <>
-  <h1>NC News</h1>
+      <Header/>
+      <Routes>
+        <Route path="/" element={<Home/>}></Route>
+        <Route path="/topics" element={<Topics/>}></Route>
+        <Route path="/users" element={<Users/>}></Route>
+        <Route path="/search" element={<Search/>}></Route>
+      </Routes>
```

Comment:
you haven't built these routes yet, so they shouldn't really appear on this ticket and branch!